src/plugins/raven/sagas/connectionFlow.js

```diff
@@ -64,23 +62,6 @@ export default function* (raven) {
     attempts += 1;
     if (attempts > 10) {
       console.error('raven::connectionFlow::exceeded_10_attempts'); // eslint-disable-line no-console
-      const userWantsToReconnect = yield swal.fire({
```

Comment:
@tbone1020 you do still want to fire the swal, you just dont need to do anything else